src/GitHub.VisualStudio/Styles/LinkDropDown.xaml

```diff
@@ -0,0 +1,83 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:cache="clr-namespace:GitHub.VisualStudio.Helpers"
+                    xmlns:ui="clr-namespace:GitHub.UI;assembly=GitHub.UI">
+    
+    <ResourceDictionary.MergedDictionaries>
+        <cache:SharedDictionaryManager Source="pack://application:,,,/GitHub.UI;component/SharedDictionary.xaml" />
+    </ResourceDictionary.MergedDictionaries>
+
+    <Style x:Key="HyperLinkToggleButton" TargetType="ToggleButton">
```

Comment:
Does it make sense then to turn the ActionLink into toggle buttons? They have the option of having the arrow, and they're already themed, would be nice if we could consolidate buttons that look like links into one.